dev/documentation/en/utilities-jobs.md

```diff
@@ -0,0 +1,89 @@
+---
+book: dev-documentation
+version: 3.4
+title: Jobs - Technical Documentation - OJS|OMP|OPS
+---
+
+# Jobs
+
+Laravel [Queues](https://laravel.com/docs/9.x/queues) can be used to create, queue and process jobs. Use jobs to prevent long-running tasks from slowing down or freezing the application.
+
+## Create a Job
+
+Create a new class for each job.
+
+```php
+/**
+ * @file jobs/email/ExampleJob.php
+ */
+namespace PKP\jobs\email;
+
+use APP\submission\Submission;
+use PKP\jobs\BaseJob;
+
+class ExampleJob extends BaseJob
+{
+    protected Submission $submission;
+
+    public function __construct(Submission $submission)
+    {
+        parent::__construct();
+
+        $this->submission = $submission;
+    }
+
+    public function handle(): void
+    {
+        // Do something...
+    }
+}
+```
+
+## Dispatch a Job
+
+When a job is dispatched, it will be added to the job queue.
+
+```php
+use PKP\jobs\email\ExampleJob;
+
+dispatch(
+    new ExampleJob($submission)
+);
+```
+
+## Process Jobs
+
+The Admin Guide describes how to [configure a job runner in production](/admin-guide/en/configure#job-runner). For local development, one of the following methods may be preferred.
+
+1. Set the `job_runner` to `On` in the config file. Jobs will be run at the end of each web request.
+2. Run `php lib/pkp/tools/jobs.php run` from the command line to run all jobs.
```

Comment:
3. Run `php lib/pkp/tools/jobs.php work` to run a daemon loop to execute jobs immediately once it dispatch to queue . use the command `php lib/pkp/tools/jobs.php work --help` to see all the possible options that can be passed to configure the worker daemon as per need . 

Also may be bit overkill for local development, should we include the setup/configuration of `supervisor` as mentioned at https://github.com/pkp/pkp-docs/issues/756#issuecomment-1225506171 as that should be the ideal approach to handle jobs in production env ?